cypress/e2e/e2e.cy.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Test suite for user authentication.
+ *
+ * This suite includes tests for the following scenarios:
+ * - Displaying the login page with necessary fields.
+ * - Logging in with valid credentials.
+ * - Showing an error message on invalid login attempts.
+ * - Logging out from the application.
+ */
+
+describe('Login', () => {
+  /**
+   * Visits the login page before each test.
+   */
+  beforeEach(() => {
+    cy.visit('http://localhost:5500/index.html');
+  });
+
+  /**
+   * Test case to verify that the user can log in with valid credentials.
+   *
+   * It checks if the logout button is visible after a successful login.
+   */
+  it('should log in with valid credentials', () => {
+    cy.get('#loginEmail').type('test@example.com');
```

Comment:
The website requires us to have a "@stud.noroff.no" email, so just change it to that 😎

Reply:
Thanks! I have fixed this now :)